concrete/src/Error/ErrorList/Error/AbstractError.php

```diff
@@ -1,45 +1,100 @@
 <?php
+
 namespace Concrete\Core\Error\ErrorList\Error;
 
 use Concrete\Core\Error\ErrorList\Field\FieldInterface;
 
-abstract class AbstractError implements ErrorInterface
+abstract class AbstractError implements HtmlAwareErrorInterface
 {
-
     /**
-     * @var FieldInterface
+     * The field associated to the error.
+     *
+     * @var \Concrete\Core\Error\ErrorList\Field\FieldInterface
      */
     protected $field;
 
     /**
-     * @return mixed
+     * Does the message contain an HTML-formatted string?
+     *
+     * @since concrete5 8.5.0a3
+     *
+     * @var bool
+     */
+    private $messageContainsHTML = false;
+
+    /**
+     * Get the error message.
+     *
+     * @return string
+     */
+    public function __toString()
+    {
+        return $this->getMessage();
+    }
+
+    /**
+     * Get the field associated to the error.
+     *
+     * @return \Concrete\Core\Error\ErrorList\Field\FieldInterface|null
      */
     public function getField()
     {
         return $this->field;
     }
 
     /**
-     * @param mixed $field
+     * Set the field associated to the error.
+     *
+     * @param \Concrete\Core\Error\ErrorList\Field\FieldInterface $field
      */
     public function setField(FieldInterface $field)
     {
         $this->field = $field;
     }
 
-    public function __toString()
+    /**
+     * {@inheritdoc}
+     *
+     * @since concrete5 8.5.0a3
+     *
+     * @see \Concrete\Core\Error\ErrorList\Error\HtmlAwareErrorInterface::messageContainsHTML()
+     */
+    public function messageContainsHTML()
     {
-        return $this->getMessage();
+        return $this->messageContainsHTML;
     }
 
+    /**
+     * Does the message contain an HTML-formatted string?
+     *
+     * @param bool $value
+     *
+     * @since concrete5 8.5.0a3
+     *
+     * @return $this
+     */
+    public function setMessageContainsHTML($value)
```

Comment:
How would you feel about `setMessageContainsHtml`?

Reply:
I agree. I personally adopt the following rule when using acronym/abbreviations:
- if the acronym is up to 2 chars: use UPPERCASE
- if the acronym has more than 2 chars: use Title case
I don't know why I used HTML instead of Html :wink:

PS: this "rule" is hightly inspired by https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-4.0/ms229043(v=vs.100)